flake/nixvim.nix

```diff
@@ -0,0 +1,44 @@
+{ inputs, self, ... }:
+{
+  imports = [
+    inputs.nixvim.flakeModules.default
+  ];
+
+  nixvim =
+    let
+      nameFunction = name: name;
```

Comment:
`lib.id`?

Reply:
Also: is that a saner default upstream?

Reply:
>lib.id?

Thanks haha 

>Also: is that a saner default upstream?

I think so

Reply:
I went with a prefixed name because I figured users may end up mixing nixvim-packages with other unrelated packages in their flake outputs :thinking: 

But maybe the simpler solution is `lib.id` by default